common/src/models/deployment.rs

```diff
@@ -29,15 +29,24 @@ pub struct Response {
     pub git_dirty: Option<bool>,
 }
 
+#[derive(Deserialize, Serialize)]
+pub struct EcsResponse {
+    pub id: String,
+    pub latest_deployment_state: EcsState,
+    pub running_id: Option<String>,
+    pub updated_at: DateTime<Utc>,
+    pub uri: String,
+    pub git_commit_id: Option<String>,
+    pub git_commit_msg: Option<String>,
+    pub git_branch: Option<String>,
+    pub git_dirty: Option<bool>,
+}
```

Comment:
Don't we want to return and print raw JSON from beta commadns before we make a frontend?

Reply:
We can, but I prefer to get some initial feedback of the frontend too. We can modify the frontend as we wish afterward. WDYT?